playwright/tests/eshop/eshop.app.spec.ts

```diff
@@ -2,140 +2,155 @@ import { test, expect } from "@playwright/test";
 
 test("eShop on Containers App Basic UI and Functionality Checks", async ({
   page,
-}) => {
-  console.log("Starting the eShop App UI tests");
+}, testInfo) => {
+  if (testInfo.retry > 0) {
+    console.log(`Retrying!`);
+  }
+
+  console.log("Starting the eShop App UI tests - Attempt: " + testInfo.retry);
 
-  console.log("Checking the necessary environment variables");
+  console.log(
```

Comment:
suggestion, make your own function for this:

```
const log = (message) => {
  console.log(`Attempt ${testInfo.retry}: ${message}`);
};
```

Then if you need to change how the log statements work, you can do that in one place.

Reply:
Added this